docs/ti.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: OP-TEE on Texas Instruments SoCs
 permalink: "/docs/ti/"
-layout: container-breadcrumb
-description: The instructions here will tell how to run OPTEE on Texas Instruments
+layout: default
```

Comment:
Why is this changed to default ? The default layout isn't as expected :) It's just the wire frame for other layouts. Sorry for the confusion.

Reply:
Oh, that was unintentionally, I had a merged conflict in that region when rebasing to master. I'll change it back, thanks for spotting this one.